modules/ROOT/pages/acl-policy.adoc

```diff
@@ -1,39 +1,39 @@
-= Access Control List Policy
+= IP Whitelist Policy
 
-The Access Control List Policy (ACL) allows you to create a general whitelist of IPs that will reach your deployed endpoints.
+The IP Whitelist policy allows you to create a general whitelist of IPs to reach your deployed endpoints.
 
-If you have an ACL policy assigned, you need to whitelist all IP addresses that are coming through your exposed endpoint.
+If you have an IP Whitelist policy assigned, you need to whitelist all IP addresses that are coming through your exposed endpoint.
 
-ACL Policy violations escalate as authentication errors for the DoS policy.
+IP Whitelist Policy violations escalate to authentication errors for the DoS policy.
 
 [NOTE]
 This source is IP based. If an attacker can spoof his source IP address, these measurements cannot prevent the attack.
 
 == Differences with API Gateway Policies
 
-The ACL policy is a whitelist for all IP addresses that go through your endpoint, and this whitelist applies to all applications. You can set up an API Gateway Whitelist policy per API list.
+The IP Whitelist policy is a whitelist for all IP addresses that go through your endpoint, and this whitelist applies to all applications. You can set up an API Gateway Whitelist policy per API list.
```

Comment:
Should define what is meant by whitelist policy -- I assume it's a policy that permits some IP addresses to access an endpoint while rejecting others, but it would be good to define it here. Saying that the "Whitelist policy is a whitelist" isn't helpful.